Source/WebKit/WebProcess/Network/webrtc/LibWebRTCNetwork.h

```diff
@@ -35,15 +35,20 @@
 #include <wtf/FunctionDispatcher.h>
 #include <wtf/TZoneMalloc.h>
 #include <wtf/UniqueRef.h>
+#include <wtf/WeakRef.h>
 
 namespace WebKit {
+class WebProcess;
 
 class LibWebRTCNetwork : private FunctionDispatcher, private IPC::MessageReceiver {
     WTF_MAKE_TZONE_ALLOCATED(LibWebRTCNetwork);
 public:
-    static UniqueRef<LibWebRTCNetwork> create() { return UniqueRef { *new LibWebRTCNetwork() }; }
+    static UniqueRef<LibWebRTCNetwork> create(WebProcess& webProcess) { return UniqueRef { *new LibWebRTCNetwork(webProcess) }; }
```

Comment:
Would be surprising if this built without updating the caller.

Also, it would be better to make the constructor public and force the caller to use `makeUniqueWithoutRefCountedCheck` (without a create) function. Otherwise, this is too error-prone now that we're ref-counted.

Reply:
You're right, it failed to build. I also had to make the inheritance from IPC::MessageReciever be public instead of private. It should build now. I also removed the create function.